turbine/src/broadcast_stage/broadcast_utils.rs

```diff
@@ -53,12 +67,22 @@ pub(super) fn recv_slot_entries(receiver: &Receiver<WorkingBankEntry>) -> Result
     }
 
     let mut serialized_batch_byte_count = serialized_size(&entries)?;
-
-    // Wait up to `ENTRY_COALESCE_DURATION` to try to coalesce entries into a 32 shred batch
+    let target_serialized_batch_byte_count: u64 = (DATA_SHREDS_PER_FEC_BLOCK
+        * ShredData::capacity(/*merkle_proof_size*/ None).unwrap())
+    .try_into()
+    .unwrap();
```

Comment:
Do we need the `try_into()` and the extra `unwrap()`?
```suggestion
    let target_serialized_batch_byte_count: u64 = (DATA_SHREDS_PER_FEC_BLOCK
        * ShredData::capacity(/*merkle_proof_size*/ None).unwrap()) as u64;
```